### PR TITLE
Added a minimal implementation of the Overture + Open Buildings 2.5D data preparation and training/inference code.

### PR DESCRIPTION
**Pull request by Akramz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +114,37 @@ Where `{location}` is one of: `bamako`, `guangdong_province`, `guatemala_departm
...
+      author={Tammy Glazer and Gilles Q. Hacheme and Akram Zaytar and Luana Marotti and Amy Michaels and Girmaw Abebe Tadesse and Kevin White and Rahul Dodhia and Andrew Zolli and Inbal Becker-Reshef and Juan M. Lavista Ferres and Caleb Robinson},
+      year={2025},
+      eprint={2511.12104},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV},
+      url={https://arxiv.org/abs/2511.12104}, 
+}
+```
+
+Paper: https://arxiv.org/abs/2511.12104
```
Comment (calebrob6) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -101,6 +114,37 @@ Where `{location}` is one of: `bamako`, `guangdong_province`, `guatemala_departm
...
+- Training configuration
+- Inference on new imagery
+
+**Note:** Requires access to Planet imagery quarterly mosaics.
+
+---
+
+## Citation
+
+Please cite the following paper if you use this code or dataset:
```
Comment (calebrob6) [inline README.md]:
```suggestion
Please cite our [paper](https://arxiv.org/abs/2511.12104) if you use this code or dataset:
```

File: scripts/infer.py [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+        img = sample["image"]
+
+        num_bands = img.shape[0]
+        if len(self.band_normalizers) != num_bands:
+            raise ValueError(
+                f"Number of band normalizers ({len(self.band_normalizers)}) "
+                f"must match number of bands ({num_bands})"
+            )
+
+        min_vals = torch.tensor(
```
Comment (calebrob6) [inline scripts/infer.py:55]: Is it possible to make `min_vals` and `max_vals` in the constructor instead of every time this is called? Might be more efficient. Also, is there a scenario where we don't just want to divide by 255?

File: scripts/infer.py [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+            rgb = f.read([1, 2, 3]).astype(np.float32)
+            rgb = np.nan_to_num(rgb, nan=0.0)
+
+        # Load Overture prior and resize to match RGB dimensions
+        with rasterio.open(overture_path) as f:
+            overture = f.read(1).astype(np.float32)  # Shape: (H, W)
+            overture = np.nan_to_num(overture, nan=0.0)
+
+            # Resize Overture to match RGB spatial dimensions if needed
+            if overture.shape != rgb.shape[1:]:
```
Comment (calebrob6) [inline scripts/infer.py:116]: Does this ever happen?

File: TRAINING.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+- `--checkpoint`: Path to trained model checkpoint (`.ckpt` file)
+- `--input-file`: Text file with `overture_path,rgb_path` pairs (one per line)
+- `--output-dir`: Directory to save predictions
+- `--gpu`: GPU ID (omit for CPU inference)
+- `--batch-size`: Number of images to process simultaneously (default: 1)
+- `--num-workers`: Data loading workers (default: 4)
+
+**Output:** Predictions saved as `{output_dir}/{quad}.tif` with 2 bands (density, height) in EPSG:3857.
+
+## Data Attribution
```
Comment (calebrob6) [inline TRAINING.md]: Do we need this seection?

File: scripts/infer.py (hunk 3 above)
Comment (calebrob6) [reply scripts/infer.py:55]: Answered my own question here -- we want to don't want to divide the overture prior by 255

File: scripts/infer.py (hunk 4 above)
Comment (Akramz) [reply scripts/infer.py:116]: Yes:
- We process and save Overture quad files to be 512×512 (same as targets).
- But for Overture's read training/inference windows to be stacked on top of Planet, they need to be resampled to the same resolution as Planet.

File: TRAINING.md (hunk 5 above)
Comment (Akramz) [reply TRAINING.md]: Removed.

**Review by calebrob6 — APPROVED** [review]
